crates/client/rpc-core/src/tests.rs

```diff
@@ -1,6 +1,20 @@
-use starknet_core::types::BlockTag;
+use std::sync::Arc;
+
+use mp_starknet::transaction::types::{BroadcastedTransactionConversionErrorWrapper, DeclareTransaction, MaxArraySize};
+use sp_core::TypedGet;
+use starknet_core::types::contract::legacy::LegacyContractClass;
+use starknet_core::types::contract::SierraClass;
+use starknet_core::types::{
+    BlockTag, BroadcastedDeclareTransactionV1, BroadcastedDeclareTransactionV2, CompressedLegacyContractClass,
+    FlattenedSierraClass,
+};
 
 use super::*;
+use crate::utils::to_declare_transaction;
+
+pub const CAIRO_1_NO_VALIDATE_ACCOUNT_COMPILED_CLASS_HASH: &str =
```

Comment:
Put that constant somewhere else and have it only once. It'll be easier to manage if we have to change it

Reply:
Ideally, it should be in `constants.rs` inside `node` but that's not accessible here so I have created a new `constants.rs` inside `rpc-core`